Project 1/styles.css

```
@@ -0,0 +1,20 @@
+.card{
+        max-width: 800px;
+        margin: 30px auto;
+        background-color: #ffffff;}
+
+img{
+    max-width: 100%;}
+    title {
+        font-family: 'Slabo', serif;
+
+}
+body{    
+    font-family: 'Roboto', sans-serif;   }
+p{margin-left: 10; margin-left: 10px;}
```

The first `margin-left` rule is missing units and should be deleted.
```suggestion
p {
  margin-left: 10px;
}
```